### PR TITLE
[Feature](tvf)(avro-jni) avro-jni add projection push down

### PR DESCRIPTION
**Pull request by DongLiang-0**
## Proposed changes
avro-jni  scanner add projection push down

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

File: fe/be-java-extensions/preload-extensions/pom.xml [hunk 1]
```diff
@@ -109,6 +109,37 @@ under the License.
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```
Comment (AshinGau) [inline fe/be-java-extensions/preload-extensions/pom.xml:112]: Why add dependencies in `preload-extensions`?

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Review by AshinGau — APPROVED** [review]

**Comment by AshinGau** [comment]
LGTM

**Review by zy-kkk — APPROVED** [review]

**Pull request merged**